docs/policies/intel-aws-instance-deny-unapproved-instance-types.md

```diff
@@ -0,0 +1,118 @@
+# Ensure aws_instance use only Intel optimized instance types (codename Ice Lake)
+
+| Provider            | Category                 |
+|---------------------|--------------------------|
+| aws                 | Infrastructure (IaaS)    |
+
+## Description
+
+Server performance is one of the most important components for enterprise applications experience. Deploying the Amazon Instances on Intel Xeon 3rd Generation Scalable processors (code-named Ice Lake) provides a seamless performance foundation.
+
+This Sentinel policy checks that the instances are configured with the recommended instance types to deliver optimal cost and performance for your business criticial servers.
+
+## Policy Results (Fail)
+
+```bash
+    trace:
+        Description:
+        ========================================================================
+                            _       _       _
+                           (_)     | |     | |
+                            _ _ __ | |_ ___| |
+                           | | '_ \| __/ _ \ |
+                           | | | | | ||  __/ |
+                           |_|_| |_|\__\___|_|
+       
+        ========================================================================
+        Name        : intel-aws-instance-deny-unapproved-instance-types.sentinel
```

Comment:
Should we have instance in this twice?

`intel-aws-instance-deny-unapproved-instance-types.sentel` 

Just asking but I don't have a better name for this either.

Reply:
aws-instance is the resource type from the provider and instance type(class) is what we're looking for in the plan. Will keep it this way to maintain pattern with other PAC modules.